cmd/zed/relationship.go

```diff
@@ -25,9 +25,11 @@ func registerRelationshipCmd(rootCmd *cobra.Command) {
 
 	relationshipCmd.AddCommand(createCmd)
 	createCmd.Flags().Bool("json", false, "output as JSON")
+	createCmd.Flags().String("caveat", "", "the caveat for the relationship, with format: 'caveat_name:{\"some\":\"context\"}'")
 
 	relationshipCmd.AddCommand(touchCmd)
 	touchCmd.Flags().Bool("json", false, "output as JSON")
+	touchCmd.Flags().String("caveat", "", "the caveat for the relationship, with format: 'caveat_name:{\"some\":\"context\"}'")
```

Comment:
This is probably a bit more readable if you use a raw string for the description so that you don't have to escape the double quotes.